view/frontend/web/js/view/cart-tagging.js

```diff
@@ -40,14 +40,22 @@ define([
   'nostojs'
 ], function (Component, customerData, nostojs) {
   'use strict';
+  let refreshRecs = false;
 
   // noinspection JSCheckFunctionSignatures
   return Component.extend({
-    initialize: function () {
+    /**
+    * Initialize Component
+    *
+    * @param config
+    */
+    initialize: function (config) {
       // noinspection JSUnresolvedFunction
       this._super();
       //noinspection JSUnusedGlobalSymbols
       this.cartTagging = customerData.get('cart-tagging');
+      //noinspection JSUnusedGlobalSymbols
+      refreshRecs = Boolean(parseInt(config.refresh))
```

Comment:
Should this be `Boolean(Number(config.refresh))` or `Boolean(config.refresh)` or do we need this `parseInt`stuff?

Reply:
`refresh` is passed as type string and Boolean will recognise any non-empty string as `true`